scalding-beam/src/test/scala/com.twitter.scalding.beam_backend/BeamBackendTests.scala

```diff
@@ -0,0 +1,49 @@
+package com.twitter.scalding.beam_backend
+
+import com.twitter.scalding.{Config, Execution, TextLine, TypedPipe}
+import org.apache.beam.sdk.Pipeline
+import org.apache.beam.sdk.options.PipelineOptionsFactory
+import org.scalatest.{BeforeAndAfter, FunSuite}
+
+class BeamBackendTests extends FunSuite with BeforeAndAfter {
+
+  private val master = "local[2]"
+  private val appName = "beam-backend-tests"
+
+  private var pipeline: Pipeline = _
+
+  before {
+    val options = PipelineOptionsFactory.create()
+    pipeline = Pipeline.create(options)
+  }
+
+  after {
+    pipeline = null
+  }
+
+  test("test") {
+    val pref = "scalding-beam/src/test/scala/com.twitter.scalding.beam_backend/BeamBackendTests.scala"
```

Comment:
Create a temp dir for this.
You can use java.nio.file.Files.createTempDirectory